### DISCUSSION TITLE
ImageMagick Supported Format

### DISCUSSION THREAD

**Discussion by programmerarunkumar** [ImageMagick/ImageMagick#5890] [Help]
Do ImageMagick support EMF,WMF,TIFF Format ?
I need to convert  EMF,WMF,TIFF Format to png 
is this possibe in IM ?

**Comment by snibgo** [comment]
The formats that IM can generally support are shown at https://www.imagemagick.org/script/formats.php . EMF and WMF are read-only. TIFF and PNG are read and write.

However, a build of IM may not support all possible formats. Use "`magick -list format`" to find what formats are supported by the IM on your computer.

**Reply by programmerarunkumar** [reply]
since EMF and WMF are read support 
can I convert the EMF and WMF to PNG ?

**Reply by programmerarunkumar** [reply]
> However, a build of IM may not support all possible formats. Use "magick -list format" to find what formats are supported by the IM on your computer.

Yes In My Machine EMF and WMF are not supported 
How can i support these format in my machine

**Reply by programmerarunkumar** [reply]
can someone clarify this ?

**Comment by snibgo** [comment]
What is your version of IM? On what platform? Did you build IM yourself? If you did, then how did you build it?

**Reply by programmerarunkumar** [reply]
platform = Linux
Version = 7.0.8

**Comment by snibgo** [comment]
You use Linux. From the page I linked https://www.imagemagick.org/script/formats.php : EMF is "Only available under Microsoft Windows."

For WMF, I suppose libwmf is required, perhaps obtained from https://download.imagemagick.org/archive/delegates/.

**Comment by amirjj** [comment]
I got the same problem. did you add any delegates for `EMF`? could you fix it on Linux?
